packages/@uppy/xhr-upload/src/index.js

```diff
@@ -181,11 +183,11 @@ module.exports = class XHRUpload extends Plugin {
     return opts
   }
 
-  addMetadata (formData, meta, opts) {
+  static addMetadata (formData, meta, opts) {
```

Comment:
Why making it `static`?

Reply:
This was from an eslint rule but I didn't think it through. Shouldn't be static here